### DISCUSSION TITLE
How to properly translate.

### DISCUSSION THREAD

**Discussion by NisheshTyagi** [OpenTTD/OpenTTD#11448] [Q&A]
Hello everyone,
I just joined this project as a translator.
But I am having problem translating the game to Hindi.
If there a string like this 
`{ORANGE}{INDUSTRY} {STRING}, {STRING}, {STRING} and {NUM} more...`
Then do i have to leave the things in {} like they already are and do should i leave GRF, OPENTTD,shortcuts and company's names to as it is or should i translate them too.
And what is the allow empty input checkbox and how to use `{P [index] "1" "other" } `.

Thank you.

**Comment by nielsmh** [comment]
I think a lot of your questions will be answered here:
https://wiki.openttd.org/en/Archive/Source/OpenTTDDevBlackBook/Format%20of%20langfiles

You should absolutely leave key terms like GRF and OpenTTD as-is. The fictitious company names for vehicle models are mostly named after several people who worked on the original Transport Tycoon game, and are also best to either leave as-is, or transliterate to the Hindi writing system.
